tensorboard/uploader/uploader_test.py

```diff
@@ -228,6 +236,138 @@ def scalar_event(tag, value):
         self.assertEqual(4 + 6, mock_client.WriteScalar.call_count)
         self.assertEqual(4 + 6, mock_rate_limiter.tick.call_count)
 
+    # Verify behavior with lots of small chunks
+    @mock.patch.object(uploader_lib, "BLOB_CHUNK_SIZE", 100)
+    def test_start_uploading_graphs(self):
+        mock_client = _create_mock_client()
+        mock_rate_limiter = mock.create_autospec(util.RateLimiter)
+        mock_blob_rate_limiter = mock.create_autospec(util.RateLimiter)
+        uploader = _create_uploader(
+            mock_client,
+            "/logs/foo",
+            rpc_rate_limiter=mock_rate_limiter,
+            blob_rpc_rate_limiter=mock_blob_rate_limiter,
+            allowed_plugins=["scalars", "graphs"],
+        )
+        uploader.create_experiment()
+
+        def graph_event(tag, value):
+            return event_pb2.Event(graph_def=value)
+
+        mock_logdir_loader = mock.create_autospec(logdir_loader.LogdirLoader)
+        mock_logdir_loader.get_run_events.side_effect = [
+            {
+                "run 1": [
+                    graph_event("1.1", bytes(950)),
```

Comment:
Please mark `srcs_version = "PY3"` for this test; in Python 2, this
evaluates to `"950"` rather than `b"\x00" * 950`.

Reply:
Done